src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/GetUnallocatedCaseService.kt

```diff
@@ -152,7 +153,11 @@ class GetUnallocatedCaseService(
     Flux.fromIterable(unallocatedCasesRepository.getCaseCountByTeam(teamCodes))
       .map { CaseCountByTeam(it.getTeamCode(), it.getCaseCount()) }
 
-  companion object {
-    private val log = LoggerFactory.getLogger(this::class.java)
-  }
+  fun getPractitionerCase(crn: String, convictionId: Long): PractitionerCase? =
```

Comment:
would this return a previously managed unallocated manager for existing cases in the db? I can only see for new cases from then on

Reply:
Yes, thought we could live with it for now as it's already happening.